### PR TITLE
Fixes JSON output in case the title field has embedded html.

### PR DESCRIPTION
**Pull request by ohvitorino**
Hi!

I made this fix because I wasn't able to embed HTML in a column's title. The generated JSON output was invalid because the quote character was not being properly escaped.

### CODE REVIEW

File: Resources/views/Column/column.html.twig [hunk 1]
```diff
@@ -21,7 +21,7 @@
         {% else %}
             "searchable": false,
         {% endif %}
-        "title": "{{ column.title|raw }}",
+        "title": "{{ column.title|raw|e('js') }}",
```
Comment (ohvitorino) [inline Resources/views/Column/column.html.twig]: Just uses the escape() function to escape the content for the json format.

**Comment by ohvitorino** [comment]
Not interesting or incorrect?

**Comment by stwe** [comment]
you mean those titles?:

```php
'title' => '<h2><i>Mein</i> HTML "Titel"</h2>'
```

**Comment by ohvitorino** [comment]
I mean using the ColumnBuilder while defining the column's title. In case a user wants to add a link as title, for example.

**Pull request closed without merging**